packages/server/src/otel/instrumentation.ts

```diff
@@ -108,7 +108,9 @@ export function httpResponseHook(
 }
 
 export function pgResponseHook(span: Span, { data }: PgResponseHookInformation): void {
-  span.setAttribute('medplum.db.rowCount', data.rowCount);
+  if (data.rowCount !== null) {
```

Comment:
The `rowCount` prop is now nullable according to the latest pg-instrumentation package, so we have to check before setting the attribute, since attributes cannot be null in OTel